docs/src/main/paradox/bootstrap/demos/kubernetes.md

```diff
@@ -1,9 +1,9 @@
-# Kubernetes
+# Kubernetes using DNS
 
-A full working example that can be deployed to kubernetes via `minikube` is in `bootstrap-demo/kubernetes-dns`.
+An example project that can be deployed to kubernetes via `minikube` is in `bootstrap-demo/kubernetes-dns`.
 
-As of Akka 2.5.15 and Akka-Management 0.18 `akka-dns` discovery mechanism can be used to deploy Akka Cluster in Kubernetes.
-This example shows how to:
+As of Akka 2.5.15 and Akka-Management 0.18 the `akka-dns` discovery mechanism can be used to deploy Akka Cluster in Kubernetes.
```

Comment:
Should `akka-dns` be changed to `async-dns` now?

Reply:
We'll want to remove the 'As of Akka 2.5.15 and Akka-Management 0.18' at some point, but might be useful for now.

Reply:
The service discovery mechanism is `akka-dns` which uses the Akka `async-dns` DNS provider

Reply:
gotcha :+1:

Reply:
I think we can remove it now. We have an explicit check for akka 2.5.15